.gitattributes

```diff
@@ -0,0 +1 @@
+dask_sphinx_theme/_version.py export-subst
```

Comment:
I'm not sure what this does. Could you provide a little more context?

Reply:
This is added automatically by versioneer. My understanding is this is the hook that triggers the `dask_sphinx_theme/_version.py` to be replaced with a static version when packaging. I'm not 100% sure how it works but it is automatically added by the `versioneer install` command.